### PR TITLE
Fixes #8955: quality-assistant should default to running merge-tests

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8955

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

File: scripts/rudder-dev/quality-assistant [hunk 1]
```diff
@@ -55,40 +55,47 @@ def clean_repo(repo):
...
+
+
+def pr_merge(repo, url):
+  redirect = ""
+  if Config.LOGLEVEL == "info" or Config.LOGLEVEL == "error":
+    redirect = " > /dev/null 2>/dev/null"
+  clean_repo(repo)
+  pr = PR(url)
+  command = "rudder-dev merge " + url + " --automatic"
+  (code, output, stderr) = shell(command + " --test" + redirect, "Trying to merge PR " + url, fail_exit=False, keep_output=True)
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/quality-assistant]: '+ " --test" + redirect' should be added to the command variable above

Comment (VinceMacBuche) [inline scripts/rudder-dev/quality-assistant]: Ok, understood you need to add the "--test" here

**Pull request merged**